client/my-sites/plugins-wpcom/plugin-types/premium-plugin.jsx

```diff
@@ -38,4 +42,16 @@ PremiumPlugin.propTypes = {
 	description: PropTypes.string.isRequired
 };
 
-export default PremiumPlugin;
+const trackClick = name => recordTracksEvent(
+	'calypso_plugin_wpcom_click',
```

Comment:
I'm not sure if this counts as a nudge, but if it is, we've been using the event name: `calypso_upgrade_nudge_cta_click` with a `cta_name` property.